packages/themes/src/style.css

```diff
@@ -0,0 +1,8 @@
+/* Import core stylesheets  */
+@import 'reset.css';
+@import 'scrollbars.css';
+@import 'fonts.css';
+
+/* Import a copy of the base variables and default theme */
+@import 'variables.css';
```

Comment:
I think we want to put these in a layer here no?

Reply:
The specificity was too low in the layer, also I think we are scrapping all layers

Reply:
Well the specificity shouldn't matter for the variables, they just need to be set